server/protocols/jmap/src/main/java/org/apache/james/jmap/model/VacationResponse.java

```diff
@@ -38,73 +39,67 @@
 @JsonDeserialize(builder = VacationResponse.Builder.class)
 public class VacationResponse {
```

Comment:
You are right

1/ there is no such creation query : just the default Vacation that you update
2/ VacationResponse is both used for retrieving the vacation object and updates, according to the spec.
However I can "split" update from retrieval even if I do not really see the benefit.